docs/business.yaml

```diff
@@ -1590,6 +1687,124 @@ components:
           addressCountry: Canada
           postalCode: V8R 2P1
           deliveryInstructions: 'Our apartment is located at the back of the building.'
+    Agm_extension:
+      type: object
+      title: AGM Extension Information Schema
+      description: Notice of AGM(Annual General Meeting) date extension allows companies to request an extension to the deadline for holding an Annual General Meeting (AGM). The longest extension granted at one time is six months.
```

Comment:
As per our conversation, the response examples can be used as is but the schema definition should exclude fields not included in the corresponding schema in the business schema repo.

The schema definitions are mostly used for API POST/PUT/PATCH operations.  The only schema that is used for response is the business schema.

Reply:
Thanks for the detailed explanation, Argus. 
I got it. Updated the schema, only use the fields from the business schema for AGM extension.